tests/integration/backward_compatible/proxy/topic_test.py

```diff
@@ -44,3 +45,24 @@ def assert_event():
 
     def test_str(self):
         self.assertTrue(str(self.topic).startswith("Topic"))
+
+    def test_publish_all(self):
+        collector = event_collector()
```

Comment:
You need to add version checks to these new tests. We are using the tests on master (tests in the backward_compatible folder) for backward compatibility tests. Without these checks, these tests would fail when they are run against the older clients. See this PR to see how you do it https://github.com/hazelcast/hazelcast-python-client/pull/584

Reply:
Yes, it should be enough for this test